pkg/services/searchV2/index.go

```diff
@@ -81,6 +207,10 @@ func (i *dashboardIndex) run(ctx context.Context) error {
 	}
 	i.logger.Info("Indexing for main org finished", "mainOrgIndexElapsed", time.Since(started), "numDashboards", len(dashboards))
 
+	go func() {
+		_ = i.initBlugeIndex(ctx, 1)
```

Comment:
This will only run once at startup (so it will not reflect changes!) but that should still be enough to evaluate at this basic level